airflow/api_connexion/endpoints/asset_endpoint.py

```diff
@@ -149,14 +149,13 @@ def get_dataset_events(
     query = apply_sorting(query, order_by, {}, allowed_attrs)
     events = session.scalars(query.offset(offset).limit(limit)).all()
     return asset_event_collection_schema.dump(
-        AssetEventCollection(dataset_events=events, total_entries=total_entries)
+        AssetEventCollection(asset_events=events, total_entries=total_entries)
     )
 
 
 def _generate_queued_event_where_clause(
     *,
     dag_id: str | None = None,
-    dataset_id: int | None = None,
```

Comment:
Shouldn’t this now have an `asset_id`?

Reply:
yes, this one is missed. let me change it now.

Reply:
oh, this was not actually used. so that's why I removed it

Reply:
OK if it’s not used we don’t need to add it.